<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="filtersBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>capture!</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="videoButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>670</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
